ec/ecresource/deploymentresource/schema_elasticsearch.go

```diff
@@ -17,51 +17,63 @@
 
 package deploymentresource
 
-import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
 
 // NewSchema returns the schema for an "ec_deployment" resource.
 func newElasticsearchResource() *schema.Resource {
 	return &schema.Resource{
 		Schema: map[string]*schema.Schema{
+			// This field is not very useful, it might be removed.
```

Comment:
I was thinking the same thing, that's why I didn't include it in the data resource 😄 Might as well just remove it?

Reply:
Yeah I think that'd be the best honestly.